spot-client/src/common/utils/window-handler.js

```diff
@@ -10,7 +10,7 @@ export default {
      * @returns {string}
      */
     getBaseUrl() {
```

Comment:
I feel like this change violates the definition, but not sure

Reply:
seems okay. window handler only exists to hide window.location related accessing for the unit tests that don't exist.